azkaban/util.py

```diff
@@ -148,6 +155,9 @@ def _convert_aliases(self):
           parser.set(section, 'url', url)
           parser.set(section, 'verify', 'false') # Backwards compatibility.
       parser.remove_section('alias')
+      return True
+    else:
```

Comment:
Can you hoist the second branch earlier?

Something like:

```python
if not parser.has_section('alias'):
  return False # Nothing to do
for alias, url in parser.items('alias'):
  # ...
return True
```

Reply:
@mtth Thanks. I fixed this by https://github.com/mtth/azkaban/pull/28/commits/bc20fe9ad30c16e54697d0f11f0c90958f1cba50